### DISCUSSION TITLE
Reccomendations about fine tunning params (chunk size, reranker top K, LLM top N).

### DISCUSSION THREAD

**Discussion by devMls** [infiniflow/ragflow#3834] [General]
Hi 

Has Anyone exprienced fine tunning this params?

I am in iterative process to finetunning in a law and compilance case:

I obtain the greatest results wtih chunksize 1024

At this moment I have 30 top N (llm) and 1000 in Top K reranker. Now I am experimenting decreseasing this params,

My process is repeat the same question with different params. Is better test with recovery test of knowegde database?

Any recommendations of what are usual ranges or initial formulas?

**Comment by KevinHuSh** [comment]
You could test it in retrieval test page of KB.
